app/celery/reporting_tasks.py

```diff
@@ -124,17 +124,28 @@ def create_nightly_notification_status_for_day(process_day, notification_type):
         f'create-nightly-notification-status-for-day task for {process_day} type {notification_type}: started'
     )
 
-    start = datetime.utcnow()
-    transit_data = fetch_notification_status_for_day(process_day=process_day, notification_type=notification_type)
-    end = datetime.utcnow()
-    current_app.logger.info(
-        f'create-nightly-notification-status-for-day task for {process_day} type {notification_type}: '
-        f'data fetched in {(end - start).seconds} seconds'
-    )
+    for (service_id,) in db.session.query(Service.id):
+        start = datetime.utcnow()
+        transit_data = fetch_status_data_for_service_and_day(
```

Comment:
Optional: I know you didn't name this, but `transit_data` is such a nebulous variable name, it doesn't tell much about what it holds. When reviewing `update_fact_notification_status` function, it took me a bit to understand what it holds. Could we rename it to `new_status_data` or something to that effect here and in `update_fact_notification_status`?

Reply:
Sure, I remember thinking the same thing.

I've gone with `new_status_rows`, where "rows" is even more specific.